{"name":"Invalid Login Scenario with wrong username and password.","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"6 990 ₽ expected [true] but found [false]","trace":"java.lang.AssertionError: 6 990 ₽ expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat pages.CatalogResultPage.checkIfProductInPriceRange(CatalogResultPage.java:65)\n\tat tests.MainTest.invalidLoginTest_InvalidUserNameInvalidPassword(MainTest.java:25)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:839)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)\n"},"stage":"finished","description":"Test Description: Login test with wrong username and wrong password.","steps":[{"name":"Open Index page","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1611089232589,"stop":1611089237151},{"name":"Open site catalog","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"category","value":"Бытовая техника"},{"name":"subcategory","value":"Кофеварки и кофемашины"}],"start":1611089237151,"stop":1611089241230},{"name":"Check if products are in price range","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"6 990 ₽ expected [true] but found [false]","trace":"java.lang.AssertionError: 6 990 ₽ expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat pages.CatalogResultPage.checkIfProductInPriceRange(CatalogResultPage.java:65)\n\tat tests.MainTest.invalidLoginTest_InvalidUserNameInvalidPassword(MainTest.java:25)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:839)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"from","value":"10000"},{"name":"to","value":"11000"}],"start":1611089241233,"stop":1611089242384}],"attachments":[{"name":"Page screenshot","source":"6483a631-3fbc-49ce-94d3-2d1138c227df-attachment","type":"image/png"},{"name":"invalidLoginTest_InvalidUserNameInvalidPassword failed and screenshot taken!","source":"4a88f0f9-552a-4bd7-bb3c-6463634931d0-attachment","type":"text/plain"}],"parameters":[],"start":1611089232588,"stop":1611089243824,"uuid":"c2638ccb-4486-4812-9cc4-ca404648c286","historyId":"ddd68e98cf51f15ad6a31e02c565cbd2","fullName":"tests.MainTest.invalidLoginTest_InvalidUserNameInvalidPassword","labels":[{"name":"package","value":"tests.MainTest"},{"name":"testClass","value":"tests.MainTest"},{"name":"testMethod","value":"invalidLoginTest_InvalidUserNameInvalidPassword"},{"name":"parentSuite","value":"Ozon Test Suite"},{"name":"suite","value":"MainTest"},{"name":"subSuite","value":"tests.MainTest"},{"name":"host","value":"MBP-Vladimir"},{"name":"thread","value":"12362@MBP-Vladimir.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Invalid username and password login test"},{"name":"feature","value":"Main Test"},{"name":"severity","value":"blocker"}],"links":[]}